composer/src/Plugin.php

```diff
@@ -55,8 +55,12 @@ public function addVagrantfile(Event $event) {
         $source = __DIR__ . '/../../.beetbox/Vagrantfile';
         $target =  $baseDir . '/Vagrantfile';
 
-        if (!file_exists($target) || md5_file($source) != md5_file($target)) {
-            copy($source, $target);
+        if (file_exists($source)) {
+            if (!file_exists($target) || md5_file($source) != md5_file($target)) {
+                copy($source, $target);
+            }
+        } elseif (file_exists($target)) {
```

Comment:
@alexdesignworks I've update to only add/update if package is available.

You'll manually need to remove the `Vagrantfile` if uninstalling the package.